[E]	manager_11@10.16.0.247	2018-10-18 11:00:00.147827 +0800	1539831600	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:00.148454 +0800	1539831600	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:00.178340 +0800	1539831600	null:null	0	CRASH REPORT Process <0.31789.21> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 11:00:00.178749 +0800	1539831600	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 11:00:00.178918 +0800	1539831600	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31734.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 11:00:00.179036 +0800	1539831600	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 11:00:05.179954 +0800	1539831605	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:05.180310 +0800	1539831605	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:05.180442 +0800	1539831605	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31791.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:00:10.181929 +0800	1539831610	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:10.182366 +0800	1539831610	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:10.182577 +0800	1539831610	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31836.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:00:15.183938 +0800	1539831615	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:15.184436 +0800	1539831615	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:15.184608 +0800	1539831615	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31887.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:00:20.185842 +0800	1539831620	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:20.186410 +0800	1539831620	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:20.216645 +0800	1539831620	null:null	0	CRASH REPORT Process <0.31983.21> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 11:00:20.217011 +0800	1539831620	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 11:00:20.217158 +0800	1539831620	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31932.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 11:00:20.217268 +0800	1539831620	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 11:00:25.217952 +0800	1539831625	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:25.218278 +0800	1539831625	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:25.218430 +0800	1539831625	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.31985.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:00:30.220038 +0800	1539831630	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:30.220516 +0800	1539831630	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:30.220738 +0800	1539831630	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32008.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:00:35.221082 +0800	1539831635	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:35.221580 +0800	1539831635	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:35.221747 +0800	1539831635	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32053.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:00:40.221918 +0800	1539831640	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:40.222439 +0800	1539831640	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:40.254527 +0800	1539831640	null:null	0	CRASH REPORT Process <0.32168.21> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 11:00:40.254814 +0800	1539831640	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 11:00:40.254943 +0800	1539831640	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32096.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 11:00:40.255053 +0800	1539831640	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 11:00:45.256009 +0800	1539831645	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:45.256394 +0800	1539831645	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:45.256538 +0800	1539831645	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32172.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:00:50.256899 +0800	1539831650	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:50.257262 +0800	1539831650	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:50.257407 +0800	1539831650	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32217.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:00:55.257980 +0800	1539831655	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:00:55.258493 +0800	1539831655	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:00:55.258664 +0800	1539831655	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32269.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:01:00.259912 +0800	1539831660	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:00.260466 +0800	1539831660	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:00.292227 +0800	1539831660	null:null	0	CRASH REPORT Process <0.32330.21> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 11:01:00.292516 +0800	1539831660	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 11:01:00.292639 +0800	1539831660	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32299.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 11:01:00.292760 +0800	1539831660	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 11:01:05.293994 +0800	1539831665	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:05.294369 +0800	1539831665	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:05.294503 +0800	1539831665	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32332.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:01:10.295925 +0800	1539831670	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:10.296254 +0800	1539831670	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:10.296382 +0800	1539831670	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32383.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:01:15.297954 +0800	1539831675	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:15.298339 +0800	1539831675	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:15.298459 +0800	1539831675	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32427.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:01:20.299913 +0800	1539831680	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:20.300249 +0800	1539831680	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:20.329924 +0800	1539831680	null:null	0	CRASH REPORT Process <0.32519.21> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 11:01:20.330221 +0800	1539831680	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 11:01:20.330395 +0800	1539831680	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32474.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 11:01:20.330552 +0800	1539831680	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 11:01:25.331962 +0800	1539831685	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:25.332234 +0800	1539831685	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:25.332392 +0800	1539831685	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32521.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:01:30.333904 +0800	1539831690	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:30.334167 +0800	1539831690	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:30.334312 +0800	1539831690	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32572.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:01:35.335952 +0800	1539831695	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:35.336263 +0800	1539831695	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:35.336399 +0800	1539831695	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32606.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:01:40.336849 +0800	1539831700	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:40.337349 +0800	1539831700	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:40.367190 +0800	1539831700	null:null	0	CRASH REPORT Process <0.32705.21> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 11:01:40.367624 +0800	1539831700	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 11:01:40.367795 +0800	1539831700	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32637.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 11:01:40.367933 +0800	1539831700	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
[E]	manager_11@10.16.0.247	2018-10-18 11:01:45.368988 +0800	1539831705	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:45.369304 +0800	1539831705	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:45.369440 +0800	1539831705	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32707.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:01:50.370913 +0800	1539831710	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:50.371227 +0800	1539831710	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:50.371386 +0800	1539831710	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.32759.21> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:01:55.372963 +0800	1539831715	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:01:55.373324 +0800	1539831715	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:01:55.373448 +0800	1539831715	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.52.22> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[E]	manager_11@10.16.0.247	2018-10-18 11:02:00.374845 +0800	1539831720	gen_server:call	0	gen_server leo_rpc_client_manager terminated with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:call/2 line 204
[E]	manager_11@10.16.0.247	2018-10-18 11:02:00.375855 +0800	1539831720	gen_server:call	0	CRASH REPORT Process leo_rpc_client_manager with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in gen_server:terminate/7 line 812
[E]	manager_11@10.16.0.247	2018-10-18 11:02:00.405746 +0800	1539831720	null:null	0	CRASH REPORT Process <0.165.22> with 1 neighbours exited with reason: {connection_error,{connection_error,econnrefused}} in gen_server:init_it/6 line 344
[E]	manager_11@10.16.0.247	2018-10-18 11:02:00.405977 +0800	1539831720	null:null	0	Supervisor leo_rpc_client_manager_1_at_13096_sup had child leo_rpc_client_manager_1_at_13096 started with leo_pod_manager:start_link(leo_rpc_client_manager_1_at_13096, 16, 16, leo_rpc_client_conn, [manager_1,"10.0.1.102",13096,0], #Fun<leo_rpc_client_sup.0.73440599>) at undefined exit with reason {connection_error,{connection_error,econnrefused}} in context start_error
[E]	manager_11@10.16.0.247	2018-10-18 11:02:00.406111 +0800	1539831720	gen_server:call	0	Supervisor leo_rpc_client_sup had child leo_rpc_client_manager started with leo_rpc_client_manager:start_link(5000) at <0.102.22> exit with reason no such process or port in call to gen_server:call(<0.1274.0>, raw_status) in context child_terminated
[W]	manager_11@10.16.0.247	2018-10-18 11:02:00.406237 +0800	1539831720	null:null	0	{module,"leo_rpc_client_sup"},{function,"start_child/3"},{line,106},{body,{{shutdown,{failed_to_start_child,leo_rpc_client_manager_1_at_13096,{connection_error,{connection_error,econnrefused}}}},{child,undefined,leo_rpc_client_manager_1_at_13096,{leo_pod_sup,start_link,[leo_rpc_client_manager_1_at_13096,16,16,leo_rpc_client_conn,[manager_1,"10.0.1.102",13096,0],#Fun<leo_rpc_client_sup.0.73440599>]},permanent,10000,supervisor,[leo_pod_sup]}}}
